ros_cross_compile/docker/sysroot.Dockerfile

```
@@ -99,6 +99,10 @@ RUN export DEBIAN_FRONTEND=noninteractive && apt-get update && \
     ./install_rosdeps.sh && \
     rm -rf /var/lib/apt/lists/*
 
+# Copy colcon defaults config and set COLCON_DEFAULTS_FILE
+COPY defaults.yaml /root
+ENV COLCON_DEFAULTS_FILE defaults.yaml
```

Maybe this instead, then we can skip a step of moving the file around
```suggestion
ENV COLCON_DEFAULTS_FILE=/root/defaults.yaml
```

---

Good idea! Done